sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -95,7 +95,12 @@
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-servicebus-jms-spring-boot-starter</artifactId>
-        <version>${azure.spring.boot.jms.integration.version}</version>
+        <version>${azure.spring.boot.integration.version}</version>
```

Comment:
The original version of `azure-servicebus-jms-spring-boot-starter` (2.3.4) is different from `project version` (2.3.3), is it OK to set to `project version` directly?

Reply:
I have already release spring-boot-starter 2.3.5 version, see https://github.com/Azure/azure-sdk-for-java/pull/15146